Dockerfile.in

```diff
@@ -23,6 +23,12 @@ COPY --from=0 /restic /bin/restic
 COPY --from=0 /restic_{NEW_RESTIC_VER} /bin/restic_{NEW_RESTIC_VER}
 COPY bin/{ARG_OS}_{ARG_ARCH}/{ARG_BIN} /{ARG_BIN}
 
+# Install "nice" and "ionice". They are dynamic library, so we need to include their dependency
+COPY --from=0 /usr/bin/nice /bin/nice
+COPY --from=0 /usr/bin/ionice /bin/ionice
+COPY --from=0 /lib/x86_64-linux-gnu/libc.so.6 /lib/x86_64-linux-gnu/
```

Comment:
We should not manually copy libc files, rather use distroless/static as the base image. That has these files.

https://github.com/GoogleContainerTools/distroless/blob/master/base/README.md

Reply:
done